connect-examples/v1/java/src/main/java/com/squareup/connectexamples/ItemManager.java

```diff
@@ -95,18 +98,21 @@ public JSONObject updateItem(String itemId) {
       return null;
     }
 
+    if (response.getStatus() != 200) {
```

Comment:
Do you want to group your if statement with the following? Response could be n=ull like the following condition right? `response != null && response.getStatus() == 200`

Reply:
I mean do you want to add `response != null` to your if statement.

Reply:
Good point, I'll add that shortly 👍